From: Sam Macbeth <sam@cliqz.com>
Date: Wed, 23 Sep 2020 11:11:16 +0200
Subject: Update strings for windows module manifest
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

IMPORTANT:
remember to copy © symbol from github
then save this file with Windows 1252 encoding
---
 browser/app/module.ver |  6 +++---
 ipc/app/module.ver     | 12 ++++++------
 2 files changed, 9 insertions(+), 9 deletions(-)

diff --git a/browser/app/module.ver b/browser/app/module.ver
index 5ef8d2a02a..264a5ac980 100644
--- a/browser/app/module.ver
+++ b/browser/app/module.ver
@@ -1,8 +1,8 @@
-WIN32_MODULE_COMPANYNAME=Mozilla Corporation
-WIN32_MODULE_COPYRIGHT=©Firefox and Mozilla Developers; available under the MPL 2 license.
+WIN32_MODULE_COMPANYNAME=Ghostery GmbH
+WIN32_MODULE_COPYRIGHT=Ghostery Private Browser; available under the MPL 2 license.
 WIN32_MODULE_PRODUCTVERSION=@MOZ_APP_WINVERSION@
 WIN32_MODULE_PRODUCTVERSION_STRING=@MOZ_APP_VERSION@
-WIN32_MODULE_TRADEMARKS=Firefox is a Trademark of The Mozilla Foundation.
+WIN32_MODULE_TRADEMARKS=Ghostery is a Trademark of Ghostery GmbH.
 WIN32_MODULE_DESCRIPTION=@MOZ_APP_DISPLAYNAME@
 WIN32_MODULE_PRODUCTNAME=@MOZ_APP_DISPLAYNAME@
 WIN32_MODULE_NAME=@MOZ_APP_DISPLAYNAME@
diff --git a/ipc/app/module.ver b/ipc/app/module.ver
index 8abbd23f0e..5e273c3db6 100644
--- a/ipc/app/module.ver
+++ b/ipc/app/module.ver
@@ -1,6 +1,6 @@
-WIN32_MODULE_COMPANYNAME=Mozilla Corporation
-WIN32_MODULE_PRODUCTVERSION=@MOZ_APP_WINVERSION@
-WIN32_MODULE_PRODUCTVERSION_STRING=@MOZ_APP_VERSION@
-WIN32_MODULE_DESCRIPTION=Plugin Container for @MOZ_APP_DISPLAYNAME@
-WIN32_MODULE_PRODUCTNAME=@MOZ_APP_DISPLAYNAME@
-WIN32_MODULE_NAME=@MOZ_APP_DISPLAYNAME@
+WIN32_MODULE_COMPANYNAME=Ghostery GmbH
+WIN32_MODULE_PRODUCTVERSION=@MOZ_APP_WINVERSION@
+WIN32_MODULE_PRODUCTVERSION_STRING=@MOZ_APP_VERSION@
+WIN32_MODULE_DESCRIPTION=Plugin Container for @MOZ_APP_DISPLAYNAME@
+WIN32_MODULE_PRODUCTNAME=@MOZ_APP_DISPLAYNAME@
+WIN32_MODULE_NAME=@MOZ_APP_DISPLAYNAME@
--
2.34.1

